### PR TITLE
Passkey automation

### PR DESCRIPTION
**Pull request by gary-archer**
Updates to automate a working passkey setup for Android and iOS.
NOTE: Before merge, revert to image:latest.

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -84,7 +85,37 @@ while [ "$(curl -k -s -o /dev/null -w ''%{http_code}'' -u "$ADMIN_USER:$ADMIN_PA
...
+if [ "$EXAMPLE_NAME" == 'haapi' ]; then
+
+  if [ "$ANDROID_FINGERPRINT" == '' ]; then
+    ANDROID_FINGERPRINT='67:60:CA:11:93:B6:5D:61:56:42:70:29:A1:10:B3:86:A8:48:C7:33:83:7B:B0:54:B0:0A:E3:E1:4A:7D:A0:A4'
+  fi
+  if [ "$ANDROID_SIGNATURE_DIGEST" == '' ]; then
+    ANDROID_SIGNATURE_DIGEST='Z2DKEZO2XWFWQnApoRCzhqhIxzODe7BUsArj4Up9oKQ='
+  fi
+  if [ "$APPLE_APP_ID" == '' ]; then
+    APPLE_APP_ID='io.curity.cat.ios.client'
```
Comment (mtrojanowski) [inline start.sh:100]: The app ID in the UI demo is actually now `io.curity.haapidemo`, but scripts in that app set the env variable anyway. Not sure if we need the defaults here. Maybe it's better to fail when these values are not set? Not sure...

File: stop.sh [hunk 2]
```diff
@@ -31,4 +28,4 @@ fi
 #
 export RUNTIME_PROTOCOL='' 
 export RUNTIME_BASE_URL=''
-docker compose --project-name $EXAMPLE_NAME down
+docker compose --project-name mobile down
```
Comment (mtrojanowski) [inline stop.sh:31]: Is the `EXAMPLE_NAME` var still required?

**Review by mtrojanowski — APPROVED** [review]

File: start.sh (hunk 1 above)
Comment (gary-archer) [reply start.sh:100]: Yeah - io.curity.cat.ios.client is used by the model SDK example so I fall back to that here. I want the newer one to be better named, without impacting the model SDK example.

Comment (mtrojanowski) [reply start.sh:100]: That makes sense. I forgot about the model SDK demo.

**Pull request merged**